sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -655,6 +661,14 @@ case class DateFormatClass(left: Expression, right: Expression, timeZoneId: Opti
  */
 @ExpressionDescription(
   usage = "_FUNC_(expr[, pattern]) - Returns the UNIX timestamp of the given time.",
+  arguments = """
+    Arguments:
+      * expr - A date/timestamp or String which is returned as a UNIX timestamp.
```

Comment:
The actual arguments are `timeExp` and `format` but args in the description are `expr` and `pattern`. This confuses slightly. Could you use the same names?